kalite/distributed/templates/distributed/base.html

```diff
@@ -171,10 +171,10 @@
                               {# Search box for responsive nav #}
                               <div class="visible-xs">
                                   <li id="search_input">
```

Comment:
Hi @natnael2012 please accept my apology for this late reply. we would like to merge your changes, but there are a few conflicts that need to be fixed. we've removed this first search bar for cleanup. can you please pull in the latest develop branch and make this change?